tic_tac_toe/game.py

```diff
@@ -24,18 +43,35 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
-
+    # if x in position
+    result = False
+    
+    if type(position) == tuple and len(position) == 2:
+        if len([x for x in position if x >= 0 and x <= 2 and type(x) == int]) == 2:
```

Comment:
This is a verbose, albeit creative solution. Not easy to read for the "why" though.